### ISSUE TITLE
Near-final updates to OHW23 pages

### ISSUE THREAD

**Issue by emiliom** [oceanhackweek/oceanhackweek.github.io#269]
Here are the pages that still need updating to OHW23. @alisonpchase @btupper and @emiliom, let's add our names next to the pages we will tackle. Thanks!

### Pages in the `ohw23` directory

- [ ] https://oceanhackweek.org/ohw23/index.html
- [ ] https://oceanhackweek.org/ohw23/schedule.html
- [ ] https://oceanhackweek.org/ohw23/logistics/index.html
- [ ] https://oceanhackweek.org/ohw23/projects/index.html
- [ ] https://oceanhackweek.org/ohw23/tutorials-index/index.html (maybe just a very brief intro paragraph/section? The rest will be a table of content that's auto-generated, I think)
- [ ] https://oceanhackweek.org/ohw23/tutorials_getting_started.html

### Pages in the `resources` directory

- [ ] https://oceanhackweek.org/resources/index.html
- [ ] https://oceanhackweek.org/resources/prep/git.html (change all references to `ohw22-proj` to `ohw23-proj`?)
- [ ] https://oceanhackweek.org/resources/prep/jupyterhub.html
- [ ] https://oceanhackweek.org/resources/prep/conda.html#installing-conda (no ohw23 updates needed per se, but we should add content here with our recommendation for installing conda/mamba locally on one's computer)

**Comment by emiliom** [comment]
Schedule page, https://oceanhackweek.org/ohw23/schedule.html: I've added the live Virtual and Seattle embedded Google Sheets. @alisonpchase can you see if anything in the top section, at the start of the page, could be improved with additional information? Add it if you can think of it, otherwise we'll call it good.

**Comment by alisonpchase** [comment]
@emiliom I think the OHW23 schedule page looks good, and I also looked at the projects/index page and looks good to me, but maybe Ben also wanted to look it over. First thing I will do now is edit resources/prep/git to change '22 to '23 references.

**Comment by alisonpchase** [comment]
@emiliom Are there specific goals for resources/index or should I take a stab at updating it? It seems we should do enough to remove the admonition sometime soon so that people are not scared off from exploring around on the resources.

**Comment by emiliom** [comment]
> Are there specific goals for resources/index or should I take a stab at updating it? 

My recommendation: do the bare minimum that seems useful enough. Maybe just briefly describe each section (say, in 2 sentences) and add a link to the entry page for each section.

> It seems we should do enough to remove the admonition sometime soon so that people are not scared off from exploring around on the resources.

I completely agree! Hopefully we can remove the admonition on the main OHW23 page soon, too.

**Comment by cathmmitchell** [comment]
I don't know where you're at with all of this, so maybe this is on your radar but didn't see anything obvious - is it possible to have a link to the resources page (specifically the preparation page) somewhere from within the OHW23 pages, so people can find all the great info about git, the hub, etc? (I knew what I was looking for and it wasn't obvious to me) I'm suggesting just writing a sentence in here: https://oceanhackweek.org/ohw23/logistics/index.html#getting-help

**Comment by emiliom** [comment]
Good point! I think this fell through the cracks, especially once we in the web team became overly familiar with the website. I'll address it in a few hours. Thanks!

**Comment by btupper** [comment]
@emiliom would you like me to take a swing at it if you are busy?  (I have managed to get back to just 10 thumbs now.)

**Comment by emiliom** [comment]
Thanks! PR #282 looks good, and I've merged it.

**Comment by emiliom** [comment]
I've added a prominent "admonition" in the main Tutorials page, pointing to the Preparation page and to the OHW22 Tutorials page. PR #283

**Comment by abkfenris** [comment]
If anyone is touching the Git/Github pages ( @ocefpaf ?), we should have [gh-scoped-creds](https://github.com/jupyterhub/gh-scoped-creds) set up on the hub now to make it easier (and more secure) than hunting down personal access tokens. I haven't gotten to test it yet.

**Comment by emiliom** [comment]
I don't think the web team had plans to do more updates to the git/github pages, so have at it! And yes, asking people to go through the personal access tokens steps is a pain.
